### DISCUSSION TITLE
Github educational benefits not recieved even after 8 days

### DISCUSSION THREAD

**Discussion by shivbera18** [community/community#145549] [GitHub Education]
### Hi everyone,

Its been 8 days but i still haven't got the benifits
![Screenshot 2024-11-25 191350](https://github.com/user-attachments/assets/ab055de1-41d4-45f1-a43d-5cdd428f0fd9)

**Comment by davevad93** [comment]
Hi @shivbera18 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @shivbera18 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
